src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -108,31 +110,20 @@ export const PublishingAccordion = ({
         })
       );
     } else if (isSuccessStatus(createPublishingRequestTicketResponse.status)) {
-      if (userCanApprovePublishingRequest) {
```

Comment:
Denne henger igjen fra gammelt av, når publiseringskurator automatisk publiserte filer når man publiserte resultatet. Nå må det gjøres i to steg, sånn at denne sjekken er overflødig